orion.py

```diff
@@ -11,8 +11,53 @@
 from pkg.logrus import SingletonLogger
 from pkg.runTest import run
 from pkg.utils import load_config
+from pkg.types import OptionMap
 
 warnings.filterwarnings("ignore", message="Unverified HTTPS request.*")
+warnings.filterwarnings(
+    "ignore", category=UserWarning, message=".*Connecting to.*verify_certs=False.*"
+)
+
+
+class MutuallyExclusiveOption(click.Option):
```

Comment:
strange that click doesn't have an inbuilt class to support mutual exclusive behaviour.